app/assets/stylesheets/partials/_gallery.scss

```diff
@@ -1,50 +1,111 @@
-.album-col {
-  height: 100%;
-}
-
-.max-height {
-  height: auto;
-  max-height: 200px;
-}
+.fotorama {
+  max-height: 80%;
 
-.img-max-height {
-  height: auto;
-  max-height: 250px;
-  max-width: 100px;
-  width: auto;
+  img {
+    &.like {
+      width: 10%;
+    }
+  }
 }
 
-.image-container {
-  height: 300px;
-  overflow: hidden;
+.album-preview {
+  margin-bottom: 10px;
   width: 100%;
 }
 
-.image-preview-container {
-  height: 200px;
-  overflow: hidden;
-  width: 100%;
-}
+.gallery {
+  &.album-preview {
+    .preview-wrap {
+      height: 100%;
+      position: relative;
+      width: 100%;
 
-.edit-image-container {
-  height: 150px;
-  overflow: hidden;
-  width: 100%;
-}
+      .caption {
+        background: rgba($black, 0.5);
+        color: $white;
+        display: inline-blocK;
+        height: 100%;
+        position: absolute;
+        text-align: center;
+        top: 0;
+        vertical-align: middle;
```

Comment:
Properties should be ordered background, color, display, height, position, text-align, top, transition, vertical-align, width